nucypher/policy/conditions/lingo.py

```diff
@@ -83,12 +83,13 @@ class InvalidExpression(ValueError):
     class ReturnValueTestSchema(CamelCaseSchema):
         comparator = fields.Str()
         value = fields.Raw(allow_none=False)  # any valid type (excludes None)
+        key = fields.Raw(allow_none=True)
 
         @post_load
         def make(self, data, **kwargs):
             return ReturnValueTest(**data)
 
-    def __init__(self, comparator: str, value):
+    def __init__(self, comparator: str, value, key: Optional[Union[int, str]] = None):
```

Comment:
Key can be a `string` or `int`...does `key` being an `int` represent an index for the result we care about eg. index from values in a tuple?

Reply:
In this function it can be agnostic - mapping of ints or an array are both compatible (additional commentary below).